asapdiscovery-data/asapdiscovery/data/operators/selectors/mcs_selector.py

```diff
@@ -13,6 +13,88 @@
 logger = logging.getLogger(__name__)
 
 
+def sort_by_mcs(
```

Comment:
@hmacdope the failing tests could be due to changes here I think this is the only thing related to the workflow tests?

Reply:
reversing the change fixed the tests on CI.  I tried to reproduce the long test time locally but failed so not sure what is happening. I'll try again tomorrow.